### ISSUE TITLE
WSL2 - [New C# Class] Illegal argument: line must be non-negative

### ISSUE THREAD

**Issue by mayconfsbrito** [KreativJos/csharpextensions#7]
I'm using VSCode with WSL2 Remote Host in my Windows10.

When I try to run **New C# Class** and insert the filename in the input text I have the problem below and the new class are not created.

For me, this problem doesn't happen in windows without wsl.

```
[2020-04-03 10:50:13.059] [exthost] [error] [kreativ-software.csharpextensions] provider FAILED
[2020-04-03 10:50:13.061] [exthost] [error] Error: Illegal argument: line must be non-negative
	at Object.t.illegalArgument (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:11:190)
	at new f (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:395:168)
	at new f (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:394:558)
	at CodeActionProvider.getInitializeFromCtorCommand (/home/mayconfsbrito/.vscode-server/extensions/kreativ-software.csharpextensions-1.3.2/out/src/codeActionProvider.js:172:63)
	at addInitalizeFromCtor (/home/mayconfsbrito/.vscode-server/extensions/kreativ-software.csharpextensions-1.3.2/out/src/codeActionProvider.js:18:30)
	at CodeActionProvider.provideCodeActions (/home/mayconfsbrito/.vscode-server/extensions/kreativ-software.csharpextensions-1.3.2/out/src/codeActionProvider.js:22:9)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:605:864
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:54:345
	at new Promise (<anonymous>)
	at Object.t.asPromise (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:54:317)
	at D.provideCodeActions (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:605:835)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:628:631
	at V._withAdapter (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:623:832)
	at V.$provideCodeActions (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:628:609)
	at p._doInvokeHandler (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:707:464)
	at p._invokeHandler (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:707:156)
	at p._receiveRequest (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:705:766)
	at p._receiveOneMessage (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:704:623)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:702:791
	at l.fire (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:48:845)
	at v.fire (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:191:325)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:870:104
	at l.fire (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:48:845)
	at v.fire (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:191:325)
	at t.PersistentProtocol._receiveMessage (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:195:717)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:192:897
	at l.fire (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:48:845)
	at p.acceptChunk (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:188:897)
	at /home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:188:249
	at Socket.t (/home/mayconfsbrito/.vscode-server/bin/0ba0ca52957102ca3527cf479571617f0de6ed50/out/vs/server/remoteExtensionHostProcess.js:197:232)
	at Socket.emit (events.js:200:13)
	at addChunk (_stream_readable.js:294:12)
	at readableAddChunk (_stream_readable.js:275:11)
	at Socket.Readable.push (_stream_readable.js:210:10)
	at TCP.onStreamRead (internal/stream_base_commons.js:166:17)
```

**Comment by KreativJos** [comment]
This seems to be resolved with the current version (1.3.6).
Could you confirm this? I have tested this with a new, very simple project, in WSL2 - Ubuntu 18.04.

**Comment by KreativJos** [comment]
Seems to work in all cases I could test (WSL2 - Ubuntu 20.04):
- Project in windows filesystem
- Project in wsl file system (user home)


Closing this issue as it seems to be resolved.
